robocopkaka/bootcamp-29-project
# #158266113 Fix error message on failed update to center

#### What does this PR do?
* fix error message showed when an update to a center fails
#### Description of Task to be completed?
* remove the image key in the `UPDATE_CENTER_FAILURE` reducer
#### How should this be manually tested?
* clone this repo
* cd into the project folder
* run `npm install` to install all dependecies
* run `npm run start:all` to start up the application
* go to `localhost:8000` on your browser
* click on `Signup`
* enter your details
* click on `Signup`
* request for admin access
* click on `Centers` on the navbar
* click on the red add button at the bottom right corner
* enter the details of the center
* click on `Add Center`
* add at least ten more centers
* click the edit icon on a center
* enter a center name that already exists and select an image
* click on `Update Center`
* you should get an error message telling you that the center name already exists
#### Any background context you want to provide?
the error message before this PR wasn't descriptive and would confuse a non -technical user
#### What are the relevant pivotal tracker stories?(if applicable)
#158266113
